lib/content/ci.yml

```
@@ -75,3 +75,33 @@ jobs:
           DOTENV_KEY: ${{ secrets.DOTENV_KEY }}
         <%/env_vault%>
         run: npm test --ignore-scripts
+
```

I think we can simplify this a bit. Instead of running a separate job, we can just add this as a step that only runs on schedule. Then we can run the specific integration tests if that exist. `npm run` actually lets you grep the existing commands pretty easily so we can do that instead of grep'ing package.json as that runs the risk (a low one) of a false positive match.

Note, I called this `test:integration` as integration is a fairly common term for the types of tests we're aiming to do here (make sure we can still integrate with 3rd party libraries). We don't need to run all the tests, we just want to run the integration slice, as the normal test run is covered above.

The whole change could then look something like this:

```suggestion
        run: npm test --ignore-scripts
     - name: Weekly Integration Tests
        if: github.event_name == 'schedule'
        run: |
          if npm run --silent | grep -q "test:integration"; then
             npm run test:integration --ignore-scripts
          else
            echo "No integration tests to run, skipping..."
          fi
```